### DISCUSSION TITLE
working on a mobile app using the patterns

### DISCUSSION THREAD

**Discussion by meirm** [danielmiessler/Fabric#242] [Ideas]
![Screenshot_20240314-120921](https://github.com/danielmiessler/fabric/assets/551713/b0d1f5d1-197b-47db-811d-f91213c10c71)

**Comment by YorkyPoo** [comment]
I have been trying to seeing if there was something equivalent to patterns on a mobile AI. I'm pretty new though.

**Reply by meirm** [reply]
the repo is https://github.com/meirm/ChatGPTApp
